scripts/build/dependencies-npm.sh

```diff
@@ -101,7 +101,7 @@ function run_install() {
   # since compiled add-ons will not work otherwise.
   npm rebuild
 
-  npm install $INSTALL_OPTS
+  npm install $INSTALL_OPTS --fetch-retries 10 --fetch-retry-maxtimeout 600000
```

Comment:
And I think a `180000` (3 minutes) max timeout should also be enough (10 minutes seems a little excessive)